integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -0,0 +1,95 @@
+import { getCsrfToken } from "../utils/csrf-token";
+
+window.addEventListener("load", () => {
+  document.getElementById("id_address")?.addEventListener("focusout", autoCompleteAddress);
+  document.getElementById("id_postcode")?.addEventListener("focusout", autoCompleteAddress);
+  document.getElementById("id_city")?.addEventListener("focusout", autoCompleteAddress);
+
+  let longitude = document.getElementById("id_longitude") as HTMLInputElement;
+  let latitude = document.getElementById("id_latitude") as HTMLInputElement;
+
+  document.getElementById("auto-fill-coordinates")?.addEventListener("input", ({ target }) => {
+    if ((target as HTMLInputElement).checked) {
+      autoCompleteAddress();
+    } else {
+      // Reset to initial value
+      longitude.value = longitude.dataset.initial;
+      latitude.value = latitude.dataset.initial;
```

Comment:
I think when the checkbox is un-checked, we don't need to perform the ajax call, we can just reset the values.